tests/utils.go

```diff
@@ -4910,6 +4910,28 @@ func GetKubeVirtConfigMap() (*k8sv1.ConfigMap, error) {
 	return cfgMap, err
 }
 
+func ReplaceKubeVirtConfigMapDataValue(key string, value string) error {
```

Comment:
This might not be needed anymore since https://github.com/kubevirt/kubevirt/pull/3542, @kbidarkar please have a look.

Reply:
Thanks @jean-edouard  , I guess I need to just use test.UpdateClusterConfigValueAndWait

ok, will check and update it.

Reply:
My bad, my function was in fact never needed. I now just used the existing one.